app/frontend/components/pages/txHistory/transactionHistory.js

```diff
@@ -62,6 +66,30 @@ const TransactionHistory = ({transactionHistory}) =>
                 {class: 'align-right'},
                 transaction.effect < 0 ? h('pre', undefined, printAda(transaction.fee)) : ''
               ),
+              h(
```

Comment:
adding those fields to the transaction history makes it too cluttered imho. I would rather keep the original structure and have the EUR/USD amounts as titles/tooltips of the the amounts in ADA triggered on hover and on click, so it works properly on mobiles, too. There is already a tooltip element for that in "common" folder